packages/jest-console/src/getConsoleOutput.ts

```diff
@@ -5,43 +5,66 @@
  * LICENSE file in the root directory of this source tree.
  */
 
-import * as path from 'path';
 import chalk = require('chalk');
-import slash = require('slash');
+import type {Config} from '@jest/types';
+import {
+  StackTraceOptions,
+  formatStackTrace,
+} from 'jest-message-util';
 import type {ConsoleBuffer} from './types';
 
 export default (
+  // @ts-ignore: cleanup after 26
   root: string,
   verbose: boolean,
   buffer: ConsoleBuffer,
+  config: Config.ProjectConfig,
```

Comment:
```suggestion
  config?: Config.ProjectConfig,
```

we have to handle this argument not being provided. People might get a newer `@jest/console` installed in their project than `jest-message-util`, and get weird errors.

Another one where we can make the breaking change in 26 🙂

Reply:
Is there a way to get a `Config.ProjectConfig` instance with default values? If not, populating a new one can be pretty tricky. 

Or maybe we can get a `StackTraceConfig` here as optional param (which is easier to populate) and get `Config.ProjectConfig` in 26?

Reply:
we don't want to create a new one. If it isn't passed, we need to deal with it missing. It might be better for 25 to just take the options `formatStackTrace` wants, and add a TODO to take project config in Jest 26.

I can play with this if you want, no need for you to spend a lot of time on it 🙂